### ISSUE TITLE
CSV export classes need to use new query methods

### ISSUE THREAD

**Issue by sunnyratilal** [easydigitaldownloads/easy-digital-downloads#6614]
The CSV export classes need to be updated to use the new query methods.

**Comment by sunnyratilal** [comment]
This is ready to be merged but is dependent on #4576 being merged into `release/3.0` first.

**Comment by sunnyratilal** [comment]
Merged.
